test/transforms.jl

```diff
@@ -228,18 +231,18 @@
     @test n.x == x
     @test n.y != y
     tₒ = revert(T, n, c)
-    @test tₒ == t
+    @test Tables.matrix(t) ≈ Tables.matrix(tₒ)
 
-    Random.seed!(42) # to reproduce the results
-    x = rand(Normal(4,3), 4000)
-    y = rand(Normal(7,5), 4000)
+    # using rng for reproducible results
+    x = rand(rng, Normal(4, 3), 4000)
+    y = rand(rng, Normal(7, 5), 4000)
     t = Table(; x, y)
     T = Scale(low=0, high=1)
     n, c = apply(T, t)
-    @test all(x -> x <= 1, n.x)
-    @test all(x -> x >= 0, n.x)
-    @test all(y -> y <= 1, n.y)
-    @test all(y -> y >= 0, n.y)
+    @test all(≤(1), n.x)
+    @test all(≥(0), n.x)
+    @test all(≤(1), n.y)
+    @test all(≥(0), n.y)
```

Comment:
Awesome style!

Reply:
Thanks, `≤(1)` it's equivalent to `x -> x ≤ 1`